src/renderer/components/top-nav/top-nav.js

```diff
@@ -273,6 +273,13 @@ export default defineComponent({
       this.showFilters = false
     },
 
+    toggleSearchFiltersDisplayed: function() {
+      this.showFilters = !this.showFilters
+      if (this.showFilters) {
+        nextTick(() => this.$refs.sortByRadio?.focus())
```

Comment:
Does this work? 
`sortByRadio` is present in `FtSearchFilters` not here

Reply:
Yes; I can't say why because I'm rusty on my Vue, but it does

Reply:
@absidue Any comment (why this works & should it be written like this

Reply:
Turns out it's not working
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/da434edb-61fd-4a9a-8e2a-1a6f137cb097)
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/bad6d3c1-03ea-432b-9277-14db8031e5d3)

Reply:
It should be moved to a mounted lifecycle hook inside ft-search-filters, then it will work and also won't need the nextTick.

Reply:
`mounted` won't work, the visibility is controlled by `v-show` not `v-if`

Reply:
Yeah, that was what my original game plan was as well, so I suppose I'll add it. No idea how it works currently without it; maybe just a bug with Angular.